### PR TITLE
Feat : Color Pallet Component On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer members, please review color pallet component
color-pallet

### CODE REVIEW

File: src/components/ColorPallet/ColorPallet.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    {
+        id: 4,
+        cssName: "purple",
+        bgProp: "bgPurple"
+    },
+    {
+        id: 5,
+        cssName: "naviBlue",
+        bgProp: "bgBlue"
```
Comment (Mohasin-Haque) [inline src/components/ColorPallet/ColorPallet.jsx]: Why you made this and how you are using it? I am not getting this part. Please explain me.

**Review by Mohasin-Haque — APPROVED** [review]
Your code is good. You can go ahead and merge it.

File: src/components/ColorPallet/ColorPallet.jsx (hunk 1 above)
Comment (Rishi05051997) [reply src/components/ColorPallet/ColorPallet.jsx]: I'm using this array for classname based on that I'm applying colors to the notes

File: src/components/ColorPallet/ColorPallet.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+.color-round {
+    border-radius: 100%;
+    height: 3rem;
+    width: 3rem;
+    margin: 0 0.5rem;
+}
+
+.red {
+    background-color: red;
```
Comment (SRUJANA-PENUGONDA13) [inline src/components/ColorPallet/ColorPallet.css]: use rgb and hashcode values for colors

File: src/components/ColorPallet/ColorPallet.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { useData } from "../../context/DataProvider"
+import { updateNote } from "../../services";
+
+const colorPalletData = [
```
Comment (SRUJANA-PENUGONDA13) [inline src/components/ColorPallet/ColorPallet.jsx]: Keep this in separate file. So that we can easy to maintain data when it becomes huge.

**Review by SRUJANA-PENUGONDA13 — COMMENTED** [review]
Code looks good. added few suggestions once go through it.

File: src/components/ColorPallet/ColorPallet.css (hunk 2 above)
Comment (Rishi05051997) [reply src/components/ColorPallet/ColorPallet.css]: Done

File: src/components/ColorPallet/ColorPallet.jsx (hunk 3 above)
Comment (Rishi05051997) [reply src/components/ColorPallet/ColorPallet.jsx]: Done

**Pull request merged**